### DISCUSSION TITLE
./test/starter_trie_test

### DISCUSSION THREAD

**Discussion by EricWui** [cmu-db/bustub#312] [Q&A] [answered]
Running main() from gmock_main.cc
[==========] Running 0 tests from 0 test suites.
[==========] 0 tests from 0 test suites ran. (0 ms total)

**Comment by GavinRay97** [comment, marked as answer]
It looks like zero tests were discovered, so they are all `DISABLED`

**Comment by EricWui** [comment]
Thank you very much!😊



---- Replied Message ----
| From | Gavin ***@***.***> |
| Date | 09/07/2022 23:35 |
| To | ***@***.***> |
| Cc | ***@***.******@***.***> |
| Subject | Re: [cmu-db/bustub] ./test/starter_trie_test (Discussion #312) |

It looks like zero tests were discovered, so they are all DISABLED
